--- a/scripts/tutorials/01_assets/run_rigid_object.py
+++ b/scripts/tutorials/01_assets/run_rigid_object.py
@@ -52,7 +52,7 @@
 
     # Create separate groups called "Origin1", "Origin2", "Origin3"
     # Each group will have a robot in it
-    origins = [[0.25, 0.25, 0.0], [-0.25, 0.25, 0.0], [0.25, -0.25, 0.0], [-0.25, -0.25, 0.0]]
+    origins = [[0.6, 0.0, 0.0], [0.2, 0.0, 0.0], [-0.2, 0.0, 0.0], [-0.6, 0.0, 0.0]]
     for i, origin in enumerate(origins):
         sim_utils.create_prim(f"/World/Origin{i}", "Xform", translation=origin)
 
